### DISCUSSION TITLE
Hangup while Dialing

### DISCUSSION THREAD

**Discussion by oliverh57** [tayler6000/pyVoIP#116] [Q&A]
Hi!

I'm a beginner to VOIP. I had a question that I couldn't figure out from the documentation - how do I hang up a call when the call is still ringing. I think hangup only works if the call state is ANSWRED rather than Dialing. 

All the best.

**Comment by oliverh57** [comment]
Update: I created a fork that has this feature.

**Reply by nicolaipre** [reply]
Unless already done - Want to make a pull request?
